cmd/root/root.go

```diff
@@ -199,7 +202,7 @@ func runRootCMD(cmd *cobra.Command, args []string) error {
 // rootExecute is the main function of the program and encapsulates the general logic
 // returns any/all errors to the caller.
 func rootExecute(ctx context.Context, urlString, dest string) error {
-	minChunkSize, err := humanize.ParseBytes(viper.GetString(config.OptMinimumChunkSize))
+	chunkSize, err := humanize.ParseBytes(viper.GetString(config.OptMinimumChunkSize))
```

Comment:
```suggestion
	chunkSize, err := humanize.ParseBytes(viper.GetString(config.OptChunkSize))
```
We should still check MinimumChunkSize and adjust ChunkSize upwards to satisfy it if it's set.

Reply:
I'm not clear why we are bothering to keep min chunk size. It seems... odd? Let's just specify the chunk size we want. Having two controls for the same thing is how we got into the mess of too many ways to control concurrency. 

If someone wants a chunk size of 1byte... it's silly but it's fine.

Reply:
because pget is already deployed to many models which set it

if they set min chunk size to 500MiB we should respect that and not have a 125MiB chunk size.

Reply:
I had wired up minchunk size and chunk size to the same value behind the scenes so they should be able to be set either flag but with --minchunk size deprecated. 

The exception is in prod where our value would continue to override (since we set with the config map).

Reply:
Oops, yes I missed that. However:

- we should not refer to the deprecated name on this line, so my suggestion above still stands
- we wire both args up to the `chunkSize` variable but we never read that variable so it's not obvious to me that it matters (instead we fetch with viper). I don't have a good mental model for what happens? I'm not sure why we bind a variable if we're never reading it?
- [pflag has docs on how to alias flag names](https://pkg.go.dev/github.com/spf13/pflag#readme-mutating-or-quot-normalizing-quot-flag-names), maybe we should do that instead?

Reply:
The aliasing is a bit weird with viper involved. Generally using an XXXVarP type instead of XXXP in cobra is the easiest way to alias as it backs to a pointer. I'm really ok either way. Your suggestions do absolutely apply in either case.

Reply:
why do we have viper, again? is it so we can configure via environment variable? it's all a bit magic, I don't understand it, and i don't find the documentation helpful in the detail (like this variable aliasing thing, I don't think the documentation defines the behaviour here).

Reply:
Viper is for the ENV binding to the flags. It's a lot of magic

Variable aliasing results in the last processed element winning i'll mkae sure the arguments cannot be both used. It should address the concerns. The ENV bits is a bit more work I'll make sure it's also clear (or at least documented) behavior